camera_cards

```diff
@@ -312,13 +321,45 @@ if [[ "${TAR}" == "YES" ]] ; then
     ## tar supplied input directory
     echo
     _report -g "Creating tar archive..."
+    _writelog "Tar archive process process started" "$(date +%FT%T)"
     tar -czvf "${AIPDIR}/objects/${MEDIAID}.tar.gz" -C "${CAMERA_CARD_DIR}" .
     _writelog "TAR ARCHIVE" "${MEDIAID}.tar.gz"
+    _writelog "Tar archive process process finished" "$(date +%FT%T)"
+fi
+
+# error reporting
+_report -b "Checking for errors during the camera_cards process..."
+if [[ -z $(cat "${TREE}") ]] || [[ -z $(cat "${MEDIAINFO_OUTPUT}") ]] || [[ -z $(cat "${EXIFTOOL_OUTPUT}") ]] || [[ -z $(cat "${FFPROBE_OUTPUT}") ]] ; then
```

Comment:
could use `[[ ! -s "$TREE" ]]` rather than cat'ing the whole file. Would be faster/cleaner.

Reply:
👍🏻 3e8dc3c8f0c3565fc409422589345afc5cdde55b